src/binding/c/op_api.txt

```diff
@@ -20,6 +20,13 @@ MPI_Op_create:
 
 MPI_Op_free:
     .desc: Frees a user-defined combination function handle
+{ -- error_check --
+    if (op_ptr->kind < MPIR_OP_KIND__USER_NONCOMMUTE) {
```

Comment:
This is nitpicking, but would `if (HANDLE_IS_BUILTIN(*op)` be somewhat more robust?

Reply:
Sounds good. Fixed.